package.json

```diff
@@ -18,6 +18,8 @@
     "babel-register": "6.14.0",
     "eslint": "3.5.0",
     "eslint-config-kentcdodds": "10.1.0",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-mocha": "^4.5.1",
```

Comment:
Thanks for pointing that out! Swapped mocha for jest in https://github.com/kentcdodds/es6-workshop/pull/12/commits/83790de9fa0bf946cfc570cafcd2dc3ac4b03f1a.